tests/test_suites/llm/sft-llama3.1-8b-1n4g-fsdp2tp1-long.sh

```
@@ -0,0 +1,47 @@
+NUM_NODES=1
+GPUS_PER_NODE=4
+STEPS_PER_RUN=250
+MAX_STEPS=250
+NUM_RUNS=$(( (MAX_STEPS + STEPS_PER_RUN - 1) / STEPS_PER_RUN ))  # Round up
+NUM_MINUTES=130
+# ===== END CONFIG =====
+
+exit_if_max_steps_reached
+
+# Run the experiment
+cd $PROJECT_ROOT
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)